### DISCUSSION TITLE
keyboard (backslash key) problem

### DISCUSSION THREAD

**Discussion by WojciechGw** [picocomputer/community#57] [Q&A] [answered]
I have a problem with typing key "\\" (backslash)
Instead of "\\" it displays "#" on screen, instead of "|" it displays "~" (with shift).

any idea ?

PS. 
1. ria and vga firmware self builded from freshly cloned, untouched sources
2. tested on two different keyboards
3. via terminal (ubuntu) everything is fine

**Comment by rumbledethumps** [comment]
Is this a EN US qwerty keyboard?

**Reply by WojciechGw** [reply]
yes
Logitech Ultra-Flat Keyboard Y-BP62a

**Reply by rumbledethumps** [reply, marked as answer]
I think that's an EN UK keyboard. It needs a custom keymap.

**Reply by WojciechGw** [reply]
I checked with a third keyboard, this time from a different manufacturer Hykker X Range, and now works as expected.
This Logitech does not have "english" keys, but apparently the design is EN UK with keycaps in EN US layout.
Thank you.

**Comment by CHiPs44** [comment]
There should be 2 Qwerty keyboards : `kbd_eng_ansi` for US and `kbd_eng_iso` for UK.
